src/views/VirtualList20Items.js

```diff
@@ -0,0 +1,36 @@
+import kind from '@enact/core/kind';
```

Comment:
I have several opinions from your PR.
1. Because there was an existing VirtualList view, it would be better to reuse it.
- The existing view did not use Sandstone/Item, I think we should modify it to use Sandstone/Item. Of course, this will affect future performance value of VirtualList, but it would have been right to use Sandstone/Item.

2.To implement the above , view must be able to use props from url parameters like screenshot test / sampler.
Ex) http://10.178.92.42:8080/VirtualList?dataSize=20&disabled=false
> You can get the value using 
const url = new URL(window.location.href);
url.searchParams.get('dataSize'));

Reply:
I used the VirtualList view for both VirtualList and VirtualList100Items and integrated Sandstone/Item into them.
I also added the functionality to use url params for both VirtualList and Items